src/pages/imageSetup/components/layout/FunnelLayout.tsx

```
@@ -1,16 +1,37 @@
+import { overlay } from 'overlay-kit';
+import { useNavigate } from 'react-router-dom';
+
 import TitleNavBar from '@/shared/components/navBar/TitleNavBar';
+import Popup from '@/shared/components/overlay/popup/Popup';
 
 interface FunnelLayoutProps {
   children: React.ReactNode;
+  currentStep: 'HouseInfo' | 'FloorPlan' | 'InteriorStyle' | 'ActivityInfo';
```

ESLint

[error] Replace `'HouseInfo'·|·'FloorPlan'·|·'InteriorStyle'·|·'ActivityInfo'` with `"HouseInfo"·|·"FloorPlan"·|·"InteriorStyle"·|·"ActivityInfo"`

(prettier/prettier)